ReactSkia/components/RSkComponentTextInput.h

```diff
@@ -24,6 +24,9 @@ class RSkComponentTextInput final : public RSkComponent {
   void OnPaint(SkCanvas *canvas) override;
  private:
   bool isInEditingMode_;
+  std::string string_{};
```

Comment:
better to have proper names for the variables for string , text and color ; 
based on its functionality